src/main/java/org/wildfly/security/auth/server/ServerAuthenticationContext.java

```diff
@@ -986,10 +986,10 @@ private void handleOne(final Callback[] callbacks, final int idx) throws IOExcep
                     try {
                         X509Certificate[] x509Certificates = X500.asX509CertificateArray(sslCallback.getSslSession().getPeerCertificates());
                         verifyEvidence(new X509PeerCertificateChainEvidence(x509Certificates));
```

Comment:
Clarification: the enclosing identity might be a securityIdentity or it might be an assigned name.